Tests/Functional/TestBundle/Command/SuccessfulCommand.php

```diff
@@ -2,10 +2,11 @@
 
 namespace JMS\JobQueueBundle\Tests\Functional\TestBundle\Command;
 
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Input\InputInterface;
 
-class SuccessfulCommand extends \Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand
+class SuccessfulCommand extends Command
```

Comment:
This command needs to return proper code as well.

Reply:
I've added the const SUCCESS here as well